src/test/java/teammates/test/driver/FieldValidator.java

```diff
@@ -0,0 +1,42 @@
+package teammates.test.driver;
+
+import teammates.common.util.Assumption;
+import teammates.common.util.SanitizationHelper;
+
+/**
+ * Used to handle the data validation aspect for test.
+ */
+public class FieldValidator extends teammates.common.util.FieldValidator {
+
+    /**
+     * Checks if the given string is a non-null non-empty string no longer than
+     * the specified length {@code maxLength}.
+     *
+     * @param fieldName
+     *            A descriptive name of the field e.g., "student name", to be
+     *            used in the return value to make the explanation more
+     *            descriptive.
+     * @param value
+     *            The string to be checked.
+     * @return An explanation of why the {@code value} is not acceptable.
+     *         Returns an empty string "" if the {@code value} is acceptable.
+     */
+    public static String getValidityInfoForSizeCappedNonEmptyString(String fieldName, int maxLength, String value) {
```

Comment:
This can be non-static to match the original. This would also fix the PMD violation.